apps/mobile/src/libs/components/gradient-slider/libs/enums/slider-config.enum.ts

```diff
@@ -0,0 +1,7 @@
+const SliderConfig = {
+	SLIDER_INITIAL_WIDTH: 0,
+	SLIDER_MAX_PERCENT: 100,
+	SLIDER_STEP_VALUE: 1,
```

Comment:
```suggestion
	INITIAL_WIDTH: 0,
	MAX_PERCENT: 100,
	STEP_VALUE: 1,
```
From the name of enum we already know, that it is for slider